src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -62,7 +66,7 @@ def filestore_mixed_type(device_facts):
         return strategies.filestore.MixedType
 
 
-def get_strategy(args):
+def get_strategy(args, devices):
```

Comment:
this is cruft Andrew, be more careful next time. Thanks.
![stepping](https://user-images.githubusercontent.com/317847/46753054-c953cc00-cc8c-11e8-9e59-f436e3f70e33.gif)

Reply:
Nope, that's necessary because I call once with ``args.devices`` and again with ``unused_devices``.

![right](https://thumbs.gfycat.com/DependentNeglectedHorsechestnutleafminer-small.gif)

Reply:
Woah I had no idea

WOAH
![woah](https://user-images.githubusercontent.com/317847/46761531-b6e48d00-cca2-11e8-99ec-0d41f7e27bc6.gif)